pinot-core/src/main/java/org/apache/pinot/core/transport/InstanceRequestHandler.java

```diff
@@ -54,16 +54,17 @@
 public class InstanceRequestHandler extends SimpleChannelInboundHandler<ByteBuf> {
   private static final Logger LOGGER = LoggerFactory.getLogger(InstanceRequestHandler.class);
 
-  // TODO: make it configurable
-  private static final int SLOW_QUERY_LATENCY_THRESHOLD_MS = 100;
+  private final long _slowQueryLatencyThresholdMs;
 
   private final TDeserializer _deserializer = new TDeserializer(new TCompactProtocol.Factory());
   private final QueryScheduler _queryScheduler;
   private final ServerMetrics _serverMetrics;
 
-  public InstanceRequestHandler(QueryScheduler queryScheduler, ServerMetrics serverMetrics) {
+  public InstanceRequestHandler(QueryScheduler queryScheduler, ServerMetrics serverMetrics,
+      long slowQueryLatencyThresholdMs) {
```

Comment:
I don't think it's a good idea to expose an internal variable like `slowQueryLatencyThresholdMs` through constructor. This variable seems to be polluting the namespace outside the class where it is not needed.